### ISSUE TITLE
Provide Example Payloads for all webhook events

### ISSUE THREAD

**Issue by adambirds** [opencollective/opencollective#4244]
I am developing an integration for [Zulip](https://zulip.com) to integrate with open collective through its webhooks but we need example payloads for all the events to be able to develop it. Please can these be provided.

**Comment by Betree** [comment]
@adambirds I see that Zulip provides [Slack-compatible webhooks](https://zulip.com/integrations/doc/slack_incoming), we could auto-detect them in our code to send events with a nice format. Could you share your Zulip instance domain with me to make sure we enable it (if it's self-hosted)?

Regarding the issue itself, I agree that adding a webhook without knowing the event's format is a poor experience. We've got an alternative proposal to solve that in https://github.com/opencollective/opencollective/issues/4071: add a button to send a test event to the webhook.

**Comment by SamWhited** [comment]
Quickly stopping by to say that #4071, while a great idea, isn't an alternative. As a devleoper gettingn started using webhooks, I still would have create something, put it up somewhere, have it print out or save the webhook payload, configure webhooks, then send a test for all theo nes I wanted before I could even start developing my actual project. This costs time and money that could be saved by having webhooks documented somewhere. Thanks!

**Comment by Betree** [comment]
@SamWhited I agree that test payloads can't replace proper documentation, but I still see them as a cheaper way of improving the situation. Especially with tools like https://webhook.site (that we could maybe surface in the interface?) that make it super-easy to setup a "debug" webhook.

**Comment by Betree** [comment]
Implemented the base system for this along with a few event payloads in https://github.com/opencollective/opencollective-frontend/pull/8029
